### DISCUSSION TITLE
Sentry spring boot logback appender - cant FW user real ip to Sentry

### DISCUSSION THREAD

**Discussion by priitserk** [getsentry/support#21] [Support]
Hi

Spring boot & Sentry & Sentry-logback

As our application is behind proxy I need to forward X-Forwarded-For value to sentry. I haven't been able to find any solution for that. Any ideas? Tried CustomBeforeSendCallback but there both MDC and Spring SecurityContextHolder is empty -> probably a filters order issue? 

Any thoughts?

**Comment by priitserk** [comment]
Man, seems Im missing a task decorator, ofcourse... will give it a try

**Reply by bruno-garcia** [reply]
Did it work?
